scale/job/models.py

```diff
@@ -3184,6 +3193,73 @@ def validate_job_type_v5(self, name, version, interface, error_mapping=None, tri
 
         return warnings
 
+    def validate_job_type_v6(self, manifest_dict, configuration_dict=None):
+        """Validates a new job type prior to attempting a save
+
+        :param manifest_dict: The Seed Manifest defining the interface for running a job of this type
+        :type manifest_dict: dict
+        :param configuration_dict: The configuration for running a job of this type, possibly None
+        :type configuration_dict: dict
+        :returns: The job type validation.
+        :rtype: :class:`job.models.JobTypeValidation`
+        """
+
+        is_valid = True
+        errors = []
+        warnings = []
+        
+        manifest = None
+        config = None
+        
+        try:
+            manifest = SeedManifest(manifest_dict, do_validate=True)
+            config = JobConfigurationV6(configuration_dict, do_validate=True). get_configuration()
+        except InvalidSeedManifestDefinition as ex:
+            is_valid = False
+            errors.append(ex.error)
+            message = 'Job type manifest invalid'
+            logger.exception(message)
+            pass
+        except InvalidJobConfiguration as ex:
+            is_valid = False
+            errors.append(ex.error)
+            message = 'Job type configuration invalid'
+            logger.exception(message)
+            pass
+
+        if config and manifest:
+            try:
+                warnings.extend(config.validate(manifest))
+            except InvalidJobConfiguration as ex:
+                is_valid = False
+                errors.append(ex.error)
+                message = 'Job type configuration invalid'
+                logger.exception(message)
+                pass
+
+        if manifest:
+            try:
```

Comment:
This validation block checking against recipe types can be removed. My new recipe work now pins recipes to a job type revision, so changing a job type never breaks a recipe now.